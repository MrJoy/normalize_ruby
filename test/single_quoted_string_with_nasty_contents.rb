'use \n to indicate newlines, and use #{hash and braces} for string interpolation.'
