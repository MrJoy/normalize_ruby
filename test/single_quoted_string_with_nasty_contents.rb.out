use \n to indicate newlines, get funky with escaping \\n embed some 'quotes' and "double quotes", maybe even a \, and use #{hash and braces} for string interpolation.
